### PR TITLE
ADS-2784 / Add doc for CMP ES support

### PR DESCRIPTION
**Pull request by olsi-qose**
Add documentation related to CMP support usage of Elasticsearch

### CODE REVIEW

File: addons/cmp/elasticsearch-support.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Support for Elasticsearch 
+
+Previous to `v2.4.1`, Magento2 offers the possibility to select the catalog search engine
```
Comment (fairlane) [inline addons/cmp/elasticsearch-support.md]: ES actually became mandatory search engine in M 2.4.0

File: addons/cmp/elasticsearch-support.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+between MySql and Elasticsearch. The following versions will use Elasticsearch by default.
+
+Nosto CMP module includes the support for Elasticsearch from `v3.0.0` with minimum
```
Comment (fairlane) [inline addons/cmp/elasticsearch-support.md]: I would rephrase this 
```suggestion
If you are using Elasticsearch as a search engine you must install Nosto CMP module version >= 3.0.0.   
```

**Review by fairlane — CHANGES_REQUESTED** [review]

**Review by mridang — APPROVED** [review]

**Review by fairlane — APPROVED** [review]
